K8s-deployment/Charts/databroker/values.yaml

```diff
@@ -0,0 +1,1302 @@
+  {{- if (contains ":" $hostPort) }}
+  {{- $port = index (splitList ":" $hostPort) 1 -}}
+  {{- end }}
+  {{- end }}
+  {{- range $index, $server := concat $host .Values.ldap.servers }}
+  auth_ldap.servers.{{ add $index 1 }} = {{ $server }}
+  {{- end }}
+  auth_ldap.port = {{ coalesce .Values.ldap.port $port }}
+  {{- if or .Values.ldap.user_dn_pattern .Values.ldap.userDnPattern }}
+  auth_ldap.user_dn_pattern = {{ coalesce .Values.ldap.user_dn_pattern .Values.ldap.userDnPattern }}
+  {{- end }}
+  {{- if .Values.ldap.basedn }}
+  auth_ldap.dn_lookup_base = {{ .Values.ldap.basedn }}
+  {{- end }}
+  {{- if .Values.ldap.uidField }}
+  auth_ldap.dn_lookup_attribute = {{ .Values.ldap.uidField }}
+  {{- end }}
+  {{- if .Values.ldap.binddn }}
+  auth_ldap.dn_lookup_bind.user_dn = {{ .Values.ldap.binddn }}
+  auth_ldap.dn_lookup_bind.password = {{ required "'ldap.bindpw' is required when 'ldap.binddn' is defined" .Values.ldap.bindpw }}
+  {{- end }}
+  {{- if .Values.ldap.tls.enabled }}
+  auth_ldap.use_ssl = {{ not .Values.ldap.tls.startTls }}
+  auth_ldap.use_starttls = {{ .Values.ldap.tls.startTls }}
+  {{- if .Values.ldap.tls.CAFilename }}
+  auth_ldap.ssl_options.cacertfile = {{ .Values.ldap.tls.certificatesMountPath }}/{{ .Values.ldap.tls.CAFilename }}
+  {{- end }}
+  {{- if .Values.ldap.tls.certFilename }}
+  auth_ldap.ssl_options.certfile = {{ .Values.ldap.tls.certificatesMountPath }}/{{ .Values.ldap.tls.certFilename }}
+  auth_ldap.ssl_options.keyfile = {{ .Values.ldap.tls.certificatesMountPath }}/{{ required "'ldap.tls.certKeyFilename' is required when 'ldap.tls.certFilename' is defined" .Values.ldap.tls.certKeyFilename }}
+  {{- end }}
+  {{- if .Values.ldap.tls.skipVerify }}
+  auth_ldap.ssl_options.verify = verify_none
+  auth_ldap.ssl_options.fail_if_no_peer_cert = false
+  {{- else if .Values.ldap.tls.verify }}
+  auth_ldap.ssl_options.verify = {{ .Values.ldap.tls.verify }}
+  {{- end }}
+  {{- end }}
+  {{- end }}
+  {{- if .Values.metrics.enabled }}
+  ## Prometheus metrics
+  ##
+  prometheus.tcp.port = 15692
+  {{- end }}
+  {{- if .Values.memoryHighWatermark.enabled }}
+  ## Memory Threshold
+  ##
+  total_memory_available_override_value = {{ include "rabbitmq.toBytes" .Values.resources.limits.memory }}
+  vm_memory_high_watermark.{{ .Values.memoryHighWatermark.type }} = {{ .Values.memoryHighWatermark.value }}
+  {{- end }}
+  management.tcp.port = 15672 # used by RS server without ssl
+  management.ssl.port = 15671
+  management.ssl.cacertfile = /opt/bitnami/rabbitmq/certs/ca_certificate.pem
+  management.ssl.certfile = /opt/bitnami/rabbitmq/certs/server_certificate.pem
+  management.ssl.keyfile = /opt/bitnami/rabbitmq/certs/server_key.pem
+  disk_free_limit.relative = 1.0
+  vm_memory_high_watermark_paging_ratio = 0.6
+  # Logging
+  log.console = true
+  log.console.level = info
+  log.file = false
+  # Monitoring
+  collect_statistics_interval = 10000
+  #prometheus.tcp.port = 15692
```

Comment:
why is this commented?

Reply:
It is already defined above in the metrics configuration. 
Removed commented line.